src/components/apps/launch/ResourceRequirements.js

```diff
@@ -111,46 +112,55 @@ const StepResourceRequirementsForm = ({
     );
 
     return (
-        <>
-            <FastField
-                id={buildDebugId(baseId, ids.RESOURCE_REQUESTS.TOOL_CPU)}
-                name={`requirements.${index}.min_cpu_cores`}
-                label={getMessage("minCPUCores")}
-                component={FormSelectField}
-            >
-                {minCPUCoreList.map((size, index) => (
-                    <MenuItem key={index} value={size}>
-                        {size}
-                    </MenuItem>
-                ))}
-            </FastField>
-            <FastField
-                id={buildDebugId(baseId, ids.RESOURCE_REQUESTS.TOOL_MEM)}
-                name={`requirements.${index}.min_memory_limit`}
-                label={getMessage("minMemory")}
-                component={FormSelectField}
-                renderValue={formatGBValue}
-            >
-                {minMemoryList.map((size, index) => (
-                    <MenuItem key={index} value={size}>
-                        {formatGBListItem(size)}
-                    </MenuItem>
-                ))}
-            </FastField>
-            <FastField
-                id={buildDebugId(baseId, ids.RESOURCE_REQUESTS.MIN_DISK_SPACE)}
-                name={`requirements.${index}.min_disk_space`}
-                label={getMessage("minDiskSpace")}
-                component={FormSelectField}
-                renderValue={formatGBValue}
-            >
-                {minDiskSpaceList.map((size, index) => (
-                    <MenuItem key={index} value={size}>
-                        {formatGBListItem(size)}
-                    </MenuItem>
-                ))}
-            </FastField>
-        </>
+        <Grid container spacing={1}>
```

Comment:
This is in a `Grid` now, when it wasn't previously.  Not sure if intentional or not.

Reply:
This is how it looked when just replacing `ExapansionPanel` with `Accordion` without `Grid`

![image](https://user-images.githubusercontent.com/1250790/86948861-cb75e380-c11b-11ea-80bc-a591f9813bbe.png)

With Grid  and changes

![image](https://user-images.githubusercontent.com/1250790/86949017-0b3ccb00-c11c-11ea-97ec-d4edb7ca33fd.png)

Well I just tested multistep and looks it needs more layout adjustments

![image](https://user-images.githubusercontent.com/1250790/86949311-74244300-c11c-11ea-94e9-ca6bc725ea3e.png)

More 🤦 (Look at the expand icon)
![image](https://user-images.githubusercontent.com/1250790/86949393-9322d500-c11c-11ea-9408-18c499bf91d5.png)

Reply:
- Changing `ExapansionPanel` to `Accordion` without my `Grid`

![image](https://user-images.githubusercontent.com/1250790/86950909-ce260800-c11e-11ea-80c6-376da30d3238.png)

- With my `Grid` changes

![image](https://user-images.githubusercontent.com/1250790/86951011-f3b31180-c11e-11ea-8e7c-88d1004f1dd4.png)